packages/fast-components-styles-msft/src/breadcrumb/index.ts

```diff
@@ -0,0 +1,42 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { ensureForegroundNormal } from "../utilities/colors";
+import { BreadcrumbClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+
+const styles: ComponentStyles<BreadcrumbClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<BreadcrumbClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+
+    return {
+        breadcrumb: {
+            padding: "4px",
+            "& ol": {
+                listStyle: "none",
+                paddingLeft: "0",

```

Comment:
need rtl for paddingLeft

Reply:
Even if its 0?

Reply:
Yep. When it switches it need to know right is 0